src/Form/Type/OrderItemDiscountAdjustmentType.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\AdminOrderCreationPlugin\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+final class OrderItemDiscountAdjustmentType extends AbstractType
```

Comment:
I am wondering, do we need two form types? We can leave it as it is and only pass `string $adjustmentType` parameter in `array $options`... They are just really the same 😄